crates/vertigo-browserdriver/src/wasm_run.js

```diff
@@ -0,0 +1,882 @@
+class MapNodes {
+    data;
+    constructor() {
+        this.data = new Map();
+    }
+    set(key, value) {
+        this.data.set(key, value);
+    }
+    getItem(key) {
+        return this.data.get(key);
+    }
+    mustGetItem(key) {
+        const item = this.data.get(key);
+        if (item === undefined) {
+            throw Error(`item not found=${key}`);
+        }
+        return item;
+    }
+    get(label, key, callback) {
+        const item = this.data.get(key);
+        if (item === undefined) {
+            console.error(`${label}->get: Item id not found = ${key}`);
+        }
+        else {
+            callback(item);
+        }
+    }
+    get2(label, key1, key2, callback) {
+        const node1 = this.data.get(key1);
+        const node2 = this.data.get(key2);
+        if (node1 === undefined) {
+            console.error(`${label}->get: Item id not found = ${key1}`);
+            return;
+        }
+        if (node2 === undefined) {
+            console.error(`${label}->get: Item id not found = ${key2}`);
+            return;
+        }
+        callback(node1, node2);
+    }
+    delete(label, key, callback) {
+        const item = this.data.get(key);
+        this.data.delete(key);
+        if (item === undefined) {
+            console.error(`${label}->delete: Item id not found = ${key}`);
+        }
+        else {
+            this.data.delete(key);
+            callback(item);
+        }
+    }
+}
+
+const createElement = (name) => {
+    if (name == "path" || name == "svg") {
+        return document.createElementNS("http://www.w3.org/2000/svg", name);
+    }
+    else {
+        return document.createElement(name);
+    }
+};
+const assertNeverCommand = (data) => {
+    console.error(data);
+    throw Error('unknown command');
+};
+// type KeydownCallbackType = (
+//     // dom_id: BigInt | null,
+//     dom_id: BigInt,         //0 - null
+//     // key: string,
+//     // code: string,
+//     alt_key: boolean,
+//     ctrl_key: boolean,
+//     shift_key: boolean,
+//     meta_key: boolean
+// ) => boolean;
```

Comment:
Can this commented code be removed completly?

Reply:
done